### PR TITLE
Feature result page

### PR DESCRIPTION
**Pull request by Pyramate**
- Modified Page result to include the list based on query search params

### CODE REVIEW

File: frontend/src/components/ButtonGroupFilmNavigation.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  // Button,
+  // Container,
+  Tabs,
+  TabList,
+  TabPanels,
+  Tab,
+  TabPanel,
+} from "@chakra-ui/react";
+
+// import { useState } from "react";
```
Comment (lchastanet) [inline frontend/src/components/ButtonGroupFilmNavigation.jsx:11]: Commented code here !

File: frontend/src/components/ButtonGroupFilmNavigation.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import CardCastingCarousel from "./Casting/CardCastingCarousel";
+import MovieVideoCarousel from "./MovieVideoCarousel";
+
+export default function ButtonGroupFilmNavigation({ movie }) {
+  // const [isActive, setIsActive] = useState("Synopsis");
+  // const buttonNames = ["Synopsis", "Casting", "Reviews", "Trailers"];
+
+  // const handleClick = (btnName) => {
+  //   setIsActive(btnName);
+  // };
```
Comment (lchastanet) [inline frontend/src/components/ButtonGroupFilmNavigation.jsx:23]: Commented code here !

File: frontend/src/components/ButtonGroupFilmNavigation.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    //           {name}
+    //         </Button>
+    //       ))}
+    //     </Container>
+
+    //     {isActive === "Synopsis" && <CardDetailSynopsis movie={movie} />}
+    //     {isActive === "Casting" && <CardCastingCarousel movie={movie} />}
+    //     {isActive === "Reviews" && <CardReviewCarousel movie={movie} />}
+    //     {isActive === "Trailers" && <MovieVideoCarousel movie={movie} />}
+    //   </div>
```
Comment (lchastanet) [inline frontend/src/components/ButtonGroupFilmNavigation.jsx:85]: Commented code here !

File: frontend/src/pages/OneMovie.jsx [hunk 4]
```diff
@@ -1,21 +1,16 @@
...
 
   return (
     <div>
       <Header />
-      <CardReviewCarousel movie={movieid} />
-      <CardCastingCarousel movie={movieid} />
-      <MovieVideoCarousel movie={movieid} />
-      <CardDetailSynopsis />
+      <ButtonGroupFilmNavigation movie={movieid} />
     </div>
```
Comment (lchastanet) [inline frontend/src/pages/OneMovie.jsx:14]: This PR seems to remove elements added during the PR that I just validated...

**Review by lchastanet — CHANGES_REQUESTED** [review]
Please remove commented code, and be sure that it's ok to remove some things...

File: frontend/src/components/ButtonGroupFilmNavigation.jsx (hunk 2 above)
Comment (Pyramate) [reply frontend/src/components/ButtonGroupFilmNavigation.jsx]: euh je crois que c'est bon ^^

File: frontend/src/components/ButtonGroupFilmNavigation.jsx (hunk 3 above)
Comment (Pyramate) [reply frontend/src/components/ButtonGroupFilmNavigation.jsx]: euh je crois que c'est bon ^^

File: frontend/src/pages/OneMovie.jsx (hunk 4 above)
Comment (Pyramate) [reply frontend/src/pages/OneMovie.jsx]: euh je crois que c'est bon ^^

**Review by lchastanet — APPROVED** [review]
Everything looks fine !

**Pull request merged**